core/java/android/ext/settings/ExtSettings.java

```diff
@@ -9,6 +9,7 @@
 
 /** @hide */
 public class ExtSettings {
+    public static final int DEFAULT_CLIPBOARD_TIMEOUT_MILLIS = 3600000;
```

Comment:
ExtSettings constants should be defined in their own file, but in this case I don't think this needs to be a constant. In fact, we could just use the public constant from frameworks/base/services/core/java/com/android/server/clipboard/ClipboardService.java

`import static com.android.server.clipboard.ClipboardService.DEFAULT_CLIPBOARD_TIMEOUT_MILLIS`

Reply:
I tried that, but for some reason I couldn't figure out, `make` complains that `com.android.server.clipboard` doesn't exist?? I'm not sure if the package should be added to `Android.bp`, but I left it for now this way. Could you verify it?

Reply:
Solved by moving the constant to a new file.